image-layout.md

```diff
@@ -53,7 +53,7 @@ afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51 ./blobs/sha256/
 ## Blobs
 
 * Object names in the `blobs` subdirectories are composed of a directory for each hash algorithm, the children of which will contain the actual content.
-* A blob, referenced with digest `<alg>:<hex>` (per [descriptor](descriptor.md#digests-and-verification)), MUST have its content stored in a file under `blobs/<alg>/<hex>`.
+* A blob, referenced with digest `<alg>:<hex>` (per [descriptor](descriptor.md#digests-and-verification)), SHOULD have its content stored in a file under `blobs/<alg>/<hex>`.
```

Comment:
I think that's weakening this too much, since it allows you to store `<alg1>:<hex1>` at `blobs/<alg2>/<hex2>` (or anywhere really).  How about:

> The content of `blobs/<alg>/<hex>` MUST match the digest `<alg>:<hex>`.